Jenkinsfile

```diff
@@ -49,7 +49,7 @@ pipeline {
         stage('Validate Poms') {
             steps {
                 retry(3) {
-                    checkout scm
+                    checkout([$class: 'GitSCM', branches: [[name: "refs/heads/${BRANCH_NAME}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'WipeWorkspace'], [$class: 'LocalBranch', localBranch: "${BRANCH_NAME}"], [$class: 'CloneOption', timeout: 30, shallow: true]], submoduleCfg: [], userRemoteConfigs: scm.userRemoteConfigs ])
```

Comment:
❓ Was the `Jenkinsfile` intentionally modified as part of a UI feature?

Reply:
There was an error when building on the CI, so the change in the Jenkinsfile was needed for it to build my PR. I believe this change still needs to be tested in Jenkins so I'm not sure if it should be included in this PR.  @TonyMorrison any thoughts?

Reply:
👍